style.css

```diff
@@ -1,38 +1,39 @@
-/*-------------------mobile version--------------*/
+/* -------------------mobile version-------------- */
 
 .body {
   display: grid;
   grid-template-columns: auto;
   height: 338.125rem;
   width: 23.438rem;
-  background-color: #E5E5E5;
+  background-color: #e5e5e5;
 }
```

Comment:
- [ ] [OPTIONAL] It appears your first section - landing page has been hugely impacted by some styles and everything is now overlapping the other, which is not good for user experience. Kindly implement necessary changes to introduce appropriate spacing, padding, widths, and heights :+1:

Reply:
done